### ISSUE TITLE
Moved rancher to a new server with ssd - not booting 

### ISSUE THREAD

**Issue by MSandro** [rancher/os#2634]
**RancherOS Version: 1.5

**Where are you running RancherOS? Proxmox basd VPS

I trying to switch to a new server on contabo.com, before I used a VPS with HDD and 6 Cores. The new Server has a SSD and 10 cores.

What I've done on the old server:
* Booted to Debian 9 live
* resized the rancher partition with gparted, now it has only about 15 GB free space
* Created a disk image with gnome-disks
* The server rebooted fine and rancher didn't gets corrupted

What I've done on the new server:
* Botted debian 9 live
* Restored the image with gnome-disks to a partition of the same size
* Rebooted and got the following error:

![screenshot_20190107-190139](https://user-images.githubusercontent.com/10937159/50785647-992ec000-12b1-11e9-923d-2efb4f3861a0.jpg)

Could someon tell me whats went wrong?

**Comment by MSandro** [comment]
No, I use Rancher OS, but to create the disk image, I've used the Debian Live image which Contabo offers me as rescue system. There is a bootable partition with my rancher os and all other data and I want to copy this partition to a new server, but restoring it there, corrupts the boot loader.

**Comment by niusmallnan** [comment]
I don't have a setup to try that, but I think this would be helpful.
https://rancher.com/docs/os/v1.x/en/about/custom-partition-layout/

You may need separate RANCHER_BOOT partition and RANCHER_STATE partitions.

For the new server, you can only copy RANCHER_STATE partition and install a RANCHER_BOOT partition.

**Comment by MSandro** [comment]
I think I've installed Rancher 1.4.0 as initial install on the old system and I see just one partition RANCHER_STATE. But I can not find a /boot folder in it ... But it is able to boot. The restored image on the new server is not able to boot.
..

**Comment by niusmallnan** [comment]
@MSandro You can get boot dir if you installed ROS into one partition RANCHER_STATE.
```
$ cd /proc/1/root/boot

$ ls -l
total 186712
-rw-r--r-- 1 root root        38 Dec 29 09:32 append
-rw-r--r-- 1 root root       142 Jan  8 09:34 global.cfg
-rw-r--r-- 1 root root  67060721 Jan  8 09:34 initrd-v1.4.2
-rw-r--r-- 1 root root 113020726 Dec 29 09:32 initrd-v1.5.0
-rw-r--r-- 1 root root      1366 Jan  8 09:34 linux-current.cfg
-rw-r--r-- 1 root root      1366 Dec 29 09:32 linux-previous.cfg
-rw-r--r-- 1 root root     12734 Jan  8 09:34 rancher.png
drwxr-xr-x 2 root root      4096 Dec 29 09:32 syslinux
-rw-r--r-- 1 root root   5531408 Jan  8 09:34 vmlinuz-4.14.73-rancher
-rw-r--r-- 1 root root   5535520 Dec 29 09:32 vmlinuz-4.14.85-rancher
```

**Comment by MSandro** [comment]
I think I'll choche a other option, I try to install rancher. Would it work if I completely copy my /var/lib/docker folder to the new server with the clean installed rancher 1.5?
